### DISCUSSION TITLE
遇到bug了

### DISCUSSION THREAD

**Discussion by NingWZ** [Meloong-Git/PCL#2049] [提问] [answered]
[错误报告-2023-06-16_7.31.39.zip](https://github.com/Hex-Dragon/PCL2/files/11763991/-2023-06-16_7.31.39.zip)
这该怎么解决

**Comment by MoYuan-CN** [comment]
仅参考意见：

根据给出的错误消息，这是一个Minecraft游戏中的错误。该错误消息表明在渲染线程中无法启动游戏，并且引发了java.lang.reflect.InvocationTargetException异常。

具体地说，Caused by部分的错误消息显示了一个java.lang.NoClassDefFoundError异常，指示找不到或无法初始化com.mojang.blaze3d.systems.RenderSystem类。这可能是由于缺少必要的依赖项或者RenderSystem类文件损坏导致的。

解决此问题的一种常见方法是尝试重新安装或更新Minecraft游戏，并确保所有相关的依赖项和所需的库文件都正确安装。您还可以尝试清除并重新安装游戏，以确保没有残留的损坏文件。~~或者也可更新显卡驱动试试看？~~

**Comment by LTCatt** [comment, marked as answer]
先更新到最新快照，使用最新快照启动，如果依然启动不了，请作为 Bug 提交 Issue

**Comment by NingWZ** [comment]
好的，谢谢啦

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
